### PR TITLE
Add deploy --app-id option

### PR DESCRIPTION
**Pull request by aguillenv**
Fixes: alantech/meta#157

### CODE REVIEW

File: avm/src/main.rs [hunk 1]
```diff
@@ -58,6 +58,7 @@ fn main() {
         .about("Deploys an .agz file to a new app with one of the deploy configs at ~/.anycloud/deploy.json")
         .arg_from_usage("<AGZ_FILE> 'Specifies the .agz file to deploy'")
         .arg_from_usage("[DEPLOY_NAME] 'Specifies the name of the deploy config to use, or the first definition if not specified'")
+        .arg_from_usage("--app-id=[APP_ID] '[Optional] Specifies the application identifier'")
```
Comment (cdmistman) [inline avm/src/main.rs:61]: I don't think it's required to put `[Optional]` here

Comment (depombo) [reply avm/src/main.rs:61]: same here https://github.com/alantech/anycloud/pull/24/files#r590443000

Comment (aguillenv) [reply avm/src/main.rs:61]: Yes, thats only for the message when we execute `--help`. I've added to make it explicit but I know it is redundant.

**Review by dfellis — APPROVED** [review]

**Review by dfellis — APPROVED** [review]

File: avm/src/main.rs (hunk 1 above)
Comment (aguillenv) [reply avm/src/main.rs]: Updated

**Review by cdmistman — APPROVED** [review]

**Pull request merged**